### ISSUE TITLE
Use Configuration Cache Compatible Way to Fetch Git Values

### ISSUE THREAD

**Issue by esafirm** [gradle/common-custom-user-data-gradle-plugin#77]
After Upgrading to Gradle 7.5, I got this error about configuration cache

<img width="1304" alt="Screen Shot 2022-07-18 at 15 45 53" src="https://user-images.githubusercontent.com/1691440/179475695-8f290f73-8593-4520-bb1c-fc39fa31d96e.png">

I think we should change the way we fetch git values from [here](https://github.com/gradle/common-custom-user-data-gradle-plugin/blob/main/src/main/java/com/gradle/CustomBuildScanEnhancements.java#L288-L29) to be configuration cache compatible

**Comment by runningcode** [comment]
Thanks for filing this. This is a duplicate of https://github.com/gradle/common-custom-user-data-gradle-plugin/issues/65 

This issue comes from the experimental flag `STABLE_CONFIGURATION_CACHE`. You can disable that flag in Gradle 7.5 and this will work fine.
Once the `STABLE_CONFIGURATION_CACHE` feature is complete, no changes will be required in the Common Custom User Data Gradle plugin.
